### DISCUSSION TITLE
(NO MORE AVAILABLE) V2B PCB's (and mini kits) available in the UK (x5)

### DISCUSSION THREAD

**Discussion by chunkysteveo** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#797] [General]
** !!! NO MORE KITS AVAILABLE !!! **

I have V2B PCBs available here in the UK for anyone in the UK, if you'd like one? (not keen on the postage costs for shipping to EU/World!)

![20240906_113021_resized](https://github.com/user-attachments/assets/a8477de9-69de-4da8-bc71-628d16ba8e52)

I also have some red or blue level shifters, and the passives (resistors and capacitors) for the data line filters - you'd just have to source the ESP, female headers if you wanted them, and the 4/6 wire connectors.

Lets say £1 plus shipping (free shipping 2nd class letter) for the PCBs (1x PCB only)?
Lets say £5 plus shipping (free shipping 2nd class large letter) for the kit  (1x PCB, 1x red **or** 1x blue level shifter, 7x 560 Ohm res, 7x 220pF cap, 1x 10K Ohm if using blue level shifter)?

**Comment by zilogZ80** [comment]
Hi, do you have any of these left at all? thanks.

**Reply by chunkysteveo** [reply]
Hi! I do actually, yeah! Have the PCB's and the kit parts too. Let me know what/which parts you're after?!

**Comment by zilogZ80** [comment]
That’s great! Will take a kit please with the red level shifter. If you can get it to me by Sat I’m happy to pay for 1st as I have some free time over the weekend to do it, but no problem if not!!

Let me know how much & I can PayPal you.

**Reply by chunkysteveo** [reply]
Hi! No problem! Just FYI you'd need the ESP8266 and the wire connectors still.

Let's call it £7 in total for 1st class. PayPal is steve@chunkymedia.co.uk 

Steve

**Reply by zilogZ80** [reply]
that's awesome, thanks - pretty sure I have an ESP in my spares already (although will double check) & I'm ordering the connectors on Amazon right now!! :)

**Reply by chunkysteveo** [reply]
> that's awesome, thanks - pretty sure I have an ESP in my spares already (although will double check) & I'm ordering the connectors on Amazon right now!! :)

You want any female headers, I have some spare?

**Reply by zilogZ80** [reply]
that would be handy actually, thanks!

**Reply by chunkysteveo** [reply]
> that would be handy actually, thanks!

Will throw them in. They just need snipping to length as they are long 20-pin ones

**Reply by chunkysteveo** [reply]
![20250305_124659](https://github.com/user-attachments/assets/b62ca633-0505-4862-a333-eef48a01db78)

Will nip to the post office now. You will be able to make the above this weekend with any luck... (ignore the switch on this one, it was an experimental board I was messing with!)

**Reply by zilogZ80** [reply]
perfect! firing up the 3D printer now :)

**Reply by chunkysteveo** [reply]
> perfect! firing up the 3D printer now :)

What you printing? I've not put mine in a case yet! 

Just posted it now 1st class - £2.60.. not far off a guess lol

**Reply by zilogZ80** [reply]
the official docs haven't been updated yet with a case for the latest board revision but there's a few different ones if you look through these discussion threads. Printing this one as it looks pretty good:
https://www.printables.com/model/907885-enclosure-wifi-module-for-bestway-lay-z-spa-hottub

let me know if you need any extra for the postage!!

**Reply by zilogZ80** [reply]
all the bits just arrived, many thanks!

**Reply by chunkysteveo** [reply]
> all the bits just arrived, many thanks!

Wow... now that was quick for a first class!! Enjoy putting it all together!

**Reply by zilogZ80** [reply]
Success! The 3D printed case looks/works great also.

<img width="1449" alt="image" src="https://github.com/user-attachments/assets/7f020c9a-5361-4ac7-b410-2906d2f85965" />
<img width="949" alt="image" src="https://github.com/user-attachments/assets/b40209fe-951f-48b8-8153-1b8409bc61b7" />

**Reply by chunkysteveo** [reply]
Ohhh yes! Looking good! I was looking at that case today - I commented on it asking about the heading being too high, some time ago - guess I never tested a print?! Haha

I can see you have headers on the board and it fits - going to print a copy off with confidence now, thanks!!

**Reply by metaljay** [reply]
> Success! The 3D printed case looks/works great also.
>

What did you use for the seal around the housing? I might print one of these too

**Reply by chunkysteveo** [reply]
> > Success! The 3D printed case looks/works great also.
> 
> What did you use for the seal around the housing? I might print one of these too

The print page says you can use a length of TPU printing filament. You could also try a long thin rubber band. I'm not too fussed about water proofing it past screwing it up, so may just pass on the seal!

**Comment by robjf-10** [comment]
Hi,
I know you offered these a while back but would you still have a pcb / red shifter kit left? I retrofitted a red shifter onto a v1 pcb and its not pretty.. Hoping to replace with a new v2 setup to try and reduce E13 errors.
Thanks, Rob

**Reply by chunkysteveo** [reply]
Hi! Yes, I've still got 4 PCBs available with the level shifter and caps/resistors.

**Reply by robjf-10** [reply]
Great, i'll email you on the above address. Cheers, Rob

**Reply by chunkysteveo** [reply]
Received, thanks!

**Reply by robjf-10** [reply]
Many thanks, arrived today.

**Comment by K19Windowmaker** [comment]
Hi are any of these still available? How do I know which shifter I need red or blue?

**Reply by chunkysteveo** [reply]
Hi, yes, still got enough for two kits. 

The level shifters - red seems to be the more reliable, but I could throw in both for an extra 50p. 

You got the ESP and are OK with soldering?

Do you need the cables too?

**Comment by K19Windowmaker** [comment]
Hi could I get one with the red level shifter, if you have the cables that would be great. But I have not looked which cables I need yet. Will try to get the pump open today to have a look. Yes I have the esp and ok with soldering


Sent from Yahoo Mail for iPad


On Saturday, April 26, 2025, 9:57 am, Stephen Ludgate ***@***.***> wrote:




Hi, yes, still got enough for two kits.

The level shifters - red seems to be the more reliable, but I could throw in both for an extra 50p.

You got the ESP and are OK with soldering?

Do you need the cables too?

**Reply by chunkysteveo** [reply]
If you find out which pin cables you have, let me know and I can add them in if you like?

**Comment by K19Windowmaker** [comment]
Hi I have checked it is a 6 wire connection. Also do you have the esp? I just checked mine and the are all esp 32.


Sent from Yahoo Mail for iPhone


On Saturday, April 26, 2025, 5:56 pm, Stephen Ludgate ***@***.***> wrote:




If you find out which pin cables you have, let me know and I can add them in if you like?

**Reply by chunkysteveo** [reply]
I don't have any ESP8266 Node MCU boards, no.

**Comment by RingerRich** [comment]
Hi there!
Do you have any kits with the red level shifter left?
Having tried (and failed) to do this with just the ESP it's become obvious that I need to do it properly and stop trying to cut corners 🤣
Thanks,
Richard.

**Reply by chunkysteveo** [reply]
Think I have one kit left. Do you need the pin cables too? 4 or 6 pin?

**Reply by RingerRich** [reply]
Brilliant, thanks. It’s a 6 pin if that’s no trouble.

I can’t see an obvious way of getting the money to you, please let me know how and I’ll send it over 🙂

**Reply by chunkysteveo** [reply]
Let me check and confirm I have the parts for the kit in the morning and we can sort money and address then.

**Comment by K19Windowmaker** [comment]
Ok can I have the kit with a red level shifter and 6 pin cables


On Sunday, April 27, 2025, 7:27 pm, Stephen Ludgate ***@***.***> wrote:




I don't have any ESP8266 Node MCU boards, no.

**Comment by K19Windowmaker** [comment]
Same here let me know how to get the money to you, is it PayPal?


Sent from Yahoo Mail for iPhone


On Sunday, April 27, 2025, 7:40 pm, RingerRich ***@***.***> wrote:




Brilliant, thanks. It’s a 6 pin if that’s no trouble.

I can’t see an obvious way of getting the money to you, please let me know how and I’ll send it over 🙂

**Reply by chunkysteveo** [reply]
Let me check and confirm I have the parts for the kit in the morning and we can sort money and address then.

**Comment by chunkysteveo** [comment]
OK @K19Windowmaker  and @RingerRich 

I have 1 kit left with a RED shifter, which I would need to give to @K19Windowmaker as they asked first. I have then my final kit, which I only have the BLUE shifter - @RingerRich - are you OK with the BLUE SHIFTER, or would you like me to order some red shifters in, which would take about 1-2 weeks to come in (perfectly OK with ordering them in for you).

Can do you both with 6-pin cables too, all you'd need to get is the NodeMCU ESP8266 board and some soldering skills.

Let me know how you both want to proceed?

**Comment by K19Windowmaker** [comment]
Hi , I would like to proceed with the kit with the red shifter and the 6 pin cables.
Thank you 


Sent from Yahoo Mail for iPhone


On Monday, April 28, 2025, 1:39 pm, Stephen Ludgate ***@***.***> wrote:




OK @K19Windowmaker and @RingerRich

I have 1 kit left with a RED shifter, which I would need to give to @K19Windowmaker as they asked first. I have then my final kit, which I only have the BLUE shifter - @RingerRich - are you OK with the BLUE SHIFTER, or would you like me to order some red shifters in, which would take about 1-2 weeks to come in (perfectly OK with ordering them in for you).

Can do you both with 6-pin cables too, all you'd need to get is the NodeMCU ESP8266 board and some soldering skills.

Let me know how you both want to proceed?

**Reply by chunkysteveo** [reply]
Ok @K19Windowmaker . Lets call your kit £7.50 1st Class delivered? My email is steve@chunkymedia.co.uk for PayPal, and you can include your postal address on the payment notes or via email?

**Reply by RingerRich** [reply]
Hi @chunkysteveo, let’s go with the blue shifter, I’m sure that will do the job. 
Thanks so much.

**Reply by chunkysteveo** [reply]
> Hi @chunkysteveo, let’s go with the blue shifter, I’m sure that will do the job. Thanks so much.

Ok @RingerRich - let's  all your kit £7.50 1st class delivered. Same as above - my PayPal email is steve@chunkymedia.co.uk and you can send your delivery name and address as a note on the payment, or via email to me. Thanks!!

(I've been using a blue one on my 6-wire Paris for over a year now without any issues)

**Reply by chunkysteveo** [reply]
> Hi @chunkysteveo, let’s go with the blue shifter, I’m sure that will do the job. Thanks so much.

Got your payment, thanks! Will post out tomorrow!

**Reply by chunkysteveo** [reply]
> Hi , I would like to proceed with the kit with the red shifter and the 6 pin cables. Thank you

Payment received @K19Windowmaker - will post out tomorrow. Thanks

**Comment by chunkysteveo** [comment]
Parcel's on their way to both of you @K19Windowmaker  and @RingerRich . Hopefully with you by Thursday or Friday.

**Reply by RingerRich** [reply]
Super, thank you!

**Comment by K19Windowmaker** [comment]
Great, thank you very much.


Sent from Yahoo Mail for iPhone


On Tuesday, April 29, 2025, 2:57 pm, Stephen Ludgate ***@***.***> wrote:




Parcel's on their way to both of you @K19Windowmaker and @RingerRich . Hopefully with you by Thursday or Friday.

**Comment by K19Windowmaker** [comment]
Hi, my kit arrived today. Thank you.


Sent from Yahoo Mail for iPad


On Monday, April 28, 2025, 5:41 pm, Stephen Ludgate ***@***.***> wrote:





Hi , I would like to proceed with the kit with the red shifter and the 6 pin cables. Thank you


Payment received @K19Windowmaker - will post out tomorrow. Thanks

**Reply by chunkysteveo** [reply]
That was nice and quick!!! Best of luck with it, get it going for the nice hot bank holiday!!

**Reply by RingerRich** [reply]
Hi @chunkysteveo
Mine also arrived yesterday, many thanks. Here’s to a fun weekend :)

**Reply by chunkysteveo** [reply]
Awesome! Best of luck with the build!
